scripts/build_industry_sector_ratios.py

```diff
@@ -374,52 +374,84 @@ def chemicals_industry():
     # putting in ammonia demand for H2 and electricity separately
 
     s_emi = idees["emi"][3:57]
-    s_out = idees["out"][8:9]
     assert s_emi.index[0] == sector
-    assert sector in str(s_out.index)
-
-    ammonia = pd.read_csv(snakemake.input.ammonia_production, index_col=0)
-
-    # ktNH3/a
-    ammonia_total = ammonia.loc[ammonia.index.intersection(eu28), str(year)].sum()
 
-    s_out -= ammonia_total
+    # convert from MtHVC/a to ktHVC/a
+    s_out = config["HVC_production_today"]*1e3
```

Comment:
Previously, `s_out` included also chlorine and methanol. Is using only HVC here correct?